src/schema/feeds.ts

```diff
@@ -1250,11 +1251,18 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       ctx,
       info,
     ) => {
+      const generator = new FeedGenerator(
+        feedClient,
+        new SimpleFeedConfigGenerator({
+          feed_config_name: FeedConfigName.PostSimilarity,
+          ...args,
```

Comment:
Just watch out `args` is few other things like `before`, `offset`... that do not belong in generator. So maybe just pass `args.post_id`?

Reply:
It is not doing anything here but just don't like when args are spread since it could do something accidentally. Again, TS `strict` would scream here.

Reply:
I'll adjust if the call now returns and I do the test cleanup :)